pkg/resources/tag.go

```diff
@@ -47,6 +47,7 @@ var tagSchema = map[string]*schema.Schema{
 	FullyQualifiedNameAttributeName: schemas.FullyQualifiedNameSchema,
 }
 
+// TODO: remove after rework of external table, materialized view, stage and table
```

Comment:
Maybe let's create a ticket for it, assign it to this todo, and set the dependencies of listed reworks on this ticket. Otherwise there's a big chance we'll just forget about it.

Reply:
Or just add this point to each resource dedicated task (so link here three issues)

Reply:
Done.